### PR TITLE
Update Download links

### PR DESCRIPTION
**Pull request by Jianqiao-Zhao**

### CODE REVIEW

File: src/helm/benchmark/augmentations/cleva_perturbation.py [hunk 1]
```diff
@@ -715,7 +726,7 @@ class MandarinToCantonesePerturbation(Perturbation):
     should_perturb_references: bool = True
 
     """ Resources """
-    SOURCE_URI: str = "https://drive.google.com/uc?id=1vljbwq0hTm7W1tz74gjPnONWJ6kSEwK2"
+    SOURCE_URL: str = "http://39.108.215.175/assets/simplified_jyutping_conversion.json"
```
Comment (lyy1994) [inline src/helm/benchmark/augmentations/cleva_perturbation.py]: You may need to rename this file to "conversion.json" in the file server to match the name in Line 734.

**Review by lyy1994 — CHANGES_REQUESTED** [review]

File: src/helm/benchmark/augmentations/cleva_perturbation.py (hunk 1 above)
Comment (Jianqiao-Zhao) [reply src/helm/benchmark/augmentations/cleva_perturbation.py]: I change the name according to your suggestion.

**Review by lyy1994 — COMMENTED** [review]
I have no other problems. I am preparing a complete run to thoroughly test this PR. Should be done within an hour.

**Review by lyy1994 — APPROVED** [review]
It can be successfully run on my side. Thanks!

**Pull request merged**